doc-content/drools-docs/src/main/asciidoc/DecisionEngine/rule-base-configuration-con.adoc

```diff
@@ -0,0 +1,70 @@
+[id='rule-base-configuration-con_{context}']
+
+= Rule base configuration
+
+{PRODUCT} contains a `RuleBaseConfiguration.java` object that you can use to configure exception handler settings, multithreaded execution, and sequential mode in the {DECISION_ENGINE}.
+
+For the rule base configuration options,
+ifdef::DM,PAM[]
+download the *{PRODUCT} [VERSION] Source Distribution* ZIP file from the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Red Hat Customer Portal] and navigate to `~/{PRODUCT_FILE}-sources/src/drools-$VERSION/drools-core/src/main/java/org/drools/core/RuleBaseConfiguration.java`.
+endif::[]
+ifdef::DROOLS,JBPM,OP[]
+see the {PRODUCT} https://github.com/kiegroup/drools/blob/7.11.x/drools-core/src/main/java/org/drools/core/RuleBaseConfiguration.java[RuleBaseConfiguration.java] page on GitHub.
+endif::[]
+
+The following rule base configuration options are available for the {DECISION_ENGINE}:
+
+drools.consequenceExceptionHandler::
+When configured, this property defines the class that manages the exceptions thrown by rule consequences. You can use this property to specify a custom exception handler for rule evaluation in the {DECISION_ENGINE}.
++
+--
+Default value: `org.drools.core.runtime.rule.impl.DefaultConsequenceExceptionHandler`
+
+[source]
+----
+drools.consequenceExceptionHandler=org.drools.core.runtime.rule.impl.MyCustomConsequenceExceptionHandler
+----
+--
+
+drools.multithreadEvaluation::
+When enabled, this property enables the {DECISION_ENGINE} to evaluate rules in parallel by dividing the Rete and Phreak pattern-matching algorithms (if applicable) in independent partitions and evaluating them in parallel. You can use this property if you have retained both Rete and Phreak algorithm functionality in your {PRODUCT} distribution.
```

Comment:
Wait, but doesn't that mean that this system property as a whole is no longer applicable? Isn't the purpose of this property to divide Rete and Phreak? Even without that last sentence, that's what the property description currently says, which was the agreed-upon description when it was previously release in the RN:

> When enabled, this property enables the {DECISION_ENGINE} to evaluate rules in parallel by dividing the Rete and Phreak pattern-matching algorithms (if applicable) in independent partitions and evaluating them in parallel.

Reply:
No, it is applicable. What I meant is that this could be reworded to: 

"When enabled, this property enables the {DECISION_ENGINE} to evaluate rules in parallel by dividing the Phreak pattern-matching algorithms (if applicable) in independent partitions and evaluating them in parallel."

It might be possible that ReteOO was still part of 7.0 so it was in the description of the property before. However now there is no ReteOO in Drools anymore, only Phreak.